infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -120,7 +120,7 @@ private void createViewRole(KubernetesClient k8sClient, String name) {
             .endMetadata()
             .withRules(
                 new KubernetesPolicyRuleBuilder()
-                    .withResources("pods")
+                    .withResources("pods", "services")
```

Comment:
Please note that it will affect only new workspaces and existing workspace still won't have permissions to list services.
It would be nice to rework it and maybe `createOrReplace` role always as we do with role bindings
https://github.com/eclipse/che/blob/a9b952297af636c8a58099e24cccc9a38929b99d/infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/project/OpenShiftWorkspaceServiceAccount.java#L75

But I'm OK to merge as is.